schemas/claim-credentials-request.yml

```diff
@@ -0,0 +1,14 @@
+$schema:  http://json-schema.org/draft-04/schema#
+title:                      "Get Credentials Request"
+description: |
+  An object with the Instance Identity Document and its corresponding signature
+type:                       object
+properties:
+  signature:
+    type: string
+    description:
+      The signature if the Identity Document, base64 encoded
```

Comment:
I don't know if I followed you on that, the `document` isn't needed because it is embedded in the signature. And if the signature is invalid, the information in the document isn't reliable. The base64 signature is what we get from Amazon.

Reply:
The document being embedded into the signature is an extension to the standard which seems to be unique to Amazon.  We should send just the pkcs7/rsa2048 file and the cleartext document.  The cleartext document should be base64 encoded to ensure that there's zero chance for JSON re-serialization errors, since any change to the file would cause verification to fail.

Reply:
Embedding the document in the signature is in the standard, just mark as optional. Amazon chose to embed it. But I see your point, will change that.